docker/install/ubuntu_install_rust.sh

```diff
@@ -20,8 +20,17 @@ set -euxo pipefail
 
 export RUSTUP_HOME=/opt/rust
 export CARGO_HOME=/opt/rust
+
 # this rustc is one supported by the installed version of rust-sgx-sdk
-curl -s -S -L https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path --profile minimal --default-toolchain stable
+HOST_ARG=
+if [ "$(getconf LONG_BIT)" == "32" ]; then
```

Comment:
why this change?

Reply:
see comment below, 32bit linux needs some prodding to install the right thing